CGPs/cgp-0128/mainnet.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+      "address": "0x0239b96D10a434a56CC9E09383077A0490cF9398",
+      "function": "deposit()",
+      "args": [],
+      "value": "10000000000000000000000000"
+    },
+    {
+      "address": "0xC668583dcbDc9ae6FA3CE46462758188adfdfC24",
```

Comment:
Shouldn't we also need the approve for CELO?

Reply:
Please see the comment above